README.md

```diff
@@ -180,9 +171,7 @@ It is where you configure your search experience, and it ties all of your compon
 </SearchProvider>
 ```
 
-For more on components, continue to the next section!
-
-While components can be handy, your search experience sometimes has requirements that don't quite fit what components provide "out of the box". In this case, it can be convenient to work directly with the "actions" and "state" provided by something we call the "SearchDriver". You can use `WithSearch` to access those in a [Render Prop](https://reactjs.org/docs/render-props.html), which gives you maximum flexibility over your experience.
+While components can be handy, a search experience can have requirements that don't quite fit what components provide "out of the box". Use `WithSearch` to access "actions" and "state" in a [Render Prop](https://reactjs.org/docs/render-props.html), giving you maximum flexibility over the experience.
```

Comment:
Side note for another time. In our docs we suggest that you can use "Components" or "work with state and actions" directly. We also refer to "state and actions" as "the headless core" in places.

What do you think about eventually calling "state and actions" / "the headless core" simply "the API", or "the low-level API".

I think it would simplify the messaging. And "low level API" is easier to wrap your head around than "headless core".

Reply:
I think so, too -- "core" has taken on a few meanings. I am in agreement.